sdk/ml/azure-ai-ml/tests/test_configs/flows/flow_with_meta/flow.meta.yaml

```diff
@@ -0,0 +1,30 @@
+name: flow_with_meta
+display_name: flow with meta
+description: flow with meta
+tags:
+  - flow
+path: ./flow.dag.yaml
+# TODO: this is not "meta"?
+additional_includes:
+  - ../additional/util.py
+
+# TODO: maybe we don't need version
+version: 1
+
+# TODO: promptflow doesn't have this concept for now
+is_deterministic: true
+
+node_variant: variant_1
+runtime: promptflow-runtime
+
+# TODO: maybe we don't need below fields
+columns_mapping:
+  groundtruth: ${data.answer}
+connections:
+  node_name:
+    connection: another_connection
+    deployment_name: deployment_name
+
+# TODO: not sure what are they for, just saw them in LoadFlowAsComponentRequest
+vm_size: Standard_D2_v2
```

Comment:
please comment them out, they are related to serveless runtime and we don't expose to customer yet. 
They should belong to run.yaml if exists

if we have vm_size in yaml, it may be 
```
reosurces:
   instance_type: Standard_D2_v2
```